metadata-ingestion/src/datahub/ingestion/source/dynamodb/dynamodb.py

```diff
@@ -212,41 +211,27 @@ def get_workunit_processors(self) -> List[Optional[MetadataWorkUnitProcessor]]:
         ]
 
     def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
-        # This is a offline call to get available region names from botocore library
-        session = boto3.Session()
-        dynamodb_regions = session.get_available_regions("dynamodb")
-        logger.info(f"region names {dynamodb_regions}")
-
-        # traverse databases in sorted order so output is consistent
-        for region in dynamodb_regions:
```

Comment:
@hsheth2 removing this will be a breaking change, which I have 2 questions about:
- We will need a point release to reflect that. How do I make sure we do that?
- How can I make that clear in the documentation? Where do you recommend that goes?